### PR TITLE
Add forceLongString option

### PR DESCRIPTION
**Pull request by jeffpyke**
Add option to output 64 bit numbers as strings rather than as numbers or longs.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -145,6 +145,7 @@ Supported options:
 
 * If you pass `--ts_proto_opt=context=true`, the Twirp services will have a Go-style `ctx` parameter, which is useful for tracing/logging/etc. if you're not using node's `async_hooks` api due to performance reasons.
 * If you pass `--ts_proto_opt=forceLong=true`, all 64 bit numbers will be parsed as instances of `Long` (using the [long](https://www.npmjs.com/package/long) library).
+* If you pass `--ts_proto_opt=forceLongString=true`, all 64 bit numbers will be outputted as strings.
```
[the remark this answers is not in the document]
Comment (jeffpyke) [reply README.markdown]: Good idea, done.

File: README.markdown [hunk 2]
```diff
@@ -241,20 +242,20 @@ Numbers are by default assumed to be plain JavaScript `numbers`. Since protobuf
...
-|  int64 | number* | Long |
-|  uint32 | number | number |
-|  uint64 | number* | Unsigned Long |
-|  sint32 | number | number |
-|  sint64 | number* | Long |
-|  fixed32 | number | number |
-|  fixed64 | number* | Unsigned Long |
-|  sfixed32 | number | number |
-|  sfixed64 | number* |  Long |
+| Protobuf number types  | Default Typescript types | `forceLong` Typescript types | `forceLongString` Typescript types |
```
[the remark this answers is not in the document]
Comment (jeffpyke) [reply README.markdown]: Done.

**Comment by jeffpyke** [comment]
Updated the test. Thanks for the review.

**Comment by stephenh** [comment]
Looks great, thanks!

**Pull request merged**